packages/pluggableWidgets/document-viewer-web/src/utils/useStyle.ts

```diff
@@ -0,0 +1,23 @@
+import { HeightUnitEnum, WidthUnitEnum } from "../../typings/DocumentViewerProps";
+import { CSSProperties, useMemo } from "react";
+
+export interface UseStyleProps {
```

Comment:
this looks like the same `Dimension` interface :thinking:

Reply:
Yes, it quite same, but I don't want to copy it 1-to-1 as I don't think padding (percent of parent) going work well with document. I would rather keep it simple.